.github/workflows/assign_project_to_issue.yml

```diff
@@ -0,0 +1,15 @@
+name: Auto Assign Issue to Project
```

Comment:
slightly more precise filename could be assign_issue_to_project.yml, so the other way around - it's the Issue (or PR) that this will be assigned to the project, not the other way around. But it's just cosmetics

Reply:
Ha, interesting point. It's actually many-to-many relationship. So question is what is assigned to what. 🤔

Reply:
I thought about this from the perspective of a programmer. 

Imagine that you have a function that has 1 input parameter and will produce project 7 as an output (7 is our project nr for now). The input is an issue and the output is particular project. So the 'element' that we are working with here as an input is the issue, so by following this thought process it is the issue that we are assigning to a project, not the other way around. 

Sorry if it's too philosophical - I just wanted to give you my reasoning :D besides - it looks like you kind of agreed with me, see line number 1 :D